### ISSUE TITLE
Add database fileds for title and description in multiple languages

### ISSUE THREAD

**Issue by derpeter** [voc/voctoweb#503]
As we have already translation and subtitles in multiple languages it would be also nice to be able to store a translated description and title. It could be displayed e.g. when the user switches the translation language.

Maybe this could also help with a language selector on the start page so people only see talks in their preferred language and talks translated to it.

It would also improve the search i guess.

**Comment by saerdnaer** [comment]
Hmm, I do not really like the corresponding feature of YouTube. In my opion, the metadata of a lecture should in the same language as the original language it was held.

**Comment by derpeter** [comment]
The request was caused by https://twitter.com/saizai/status/1345706282024628224 this request.

I agree there should be clear what is the ordinial metadata. But If a person only speaking e.g. spanisch browsing media.ccc.de german meta data will not be of much use while a spanisch translation of those would be.

**Comment by v0tti** [comment]
While I also dislike this feature on YouTube, I can see the accessibility benefit.
To do this we would also need a translation of the metadata. Since it is written by the speaker we should obtain from translating it on our own without the explicit agreement from the speaker, as a translation can potentially also change meaning.
The simplest way to achieve this is by getting the data from the source. If it is helpful from a accessibility endpoint, translation support should first be implemented in Frab and the Fahrplan. AFAIK pretalx already supports multiple languages (at least for the room names), but I don't know if we can find out what the original language of any talk is.

**Comment by derpeter** [comment]
i fully agree. And this should only be an option. I assume many speakers would at least be able to provide 2 languages them self and the lingo team might help here to. If we trust them to translate the talk live we can probably also trust them with translating the description.

**Comment by saerdnaer** [comment]
Related: https://github.com/voc/voctoweb/issues/399
